types/remarkable/index.d.ts

```diff
@@ -2,8 +2,11 @@
 // Project: https://github.com/jonschlinkert/remarkable
 // Definitions by: makepost <https://github.com/makepost>
 //                 Richard Lea <https://github.com/chigix>
+//                 Lilian Saget-Lethias <https://github.com/bios21>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
 import Remarkable = require("./lib");
 
 export = Remarkable;
+export as namespace Remarkable;
```

Comment:
Why create another export? If namespace export is better, would not it be necessary to remove the previous direct export? What's more, the import usage in test file is not revised for this newly added namespace export.

Reply:
Like React types for example, it helps when various kind of imports are made + it does not have conflict with the previous declaration.

Because of that, test should not be changed and already cover this change because the behavior should remain the same. (useful when using `esModuleInterop` in tsconfig options for example).